src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -523,9 +518,8 @@ public BytesCapsule getDynamic(byte[] word){
     } else {
       try {
         bytesCapsule = getDynamicPropertiesStore().get(word);
-      } catch (BadItemException e) {
-        bytesCapsule = null;
-      } catch (ItemNotFoundException e) {
+      } catch (BadItemException | ItemNotFoundException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either log or rethrow this exception. ([squid:S1166](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1166))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1667&issues=AWbODGoHLpqqSUlcKsP8&open=AWbODGoHLpqqSUlcKsP8)

[ISSUE_KEY]: AWbODGoHLpqqSUlcKsP8